src/pl/ogladajanime/src/eu/kanade/tachiyomi/animeextension/pl/ogladajanime/OgladajAnime.kt

```diff
@@ -0,0 +1,319 @@
+
+    // =============================== Latest ===============================
+
+    override fun latestUpdatesRequest(page: Int): Request = GET("$baseUrl/search/new/$page")
+
+    override fun latestUpdatesSelector(): String = popularAnimeSelector()
+
+    override fun latestUpdatesNextPageSelector(): String = popularAnimeNextPageSelector()
+
+    override fun latestUpdatesFromElement(element: Element): SAnime = popularAnimeFromElement(element)
+
+    // =============================== Search ===============================
+
+//    override suspend fun getSearchAnime(
+//        page: Int,
+//        query: String,
+//        filters: AnimeFilterList,
+//    ): AnimesPage {
+//        return client.newCall(searchAnimeRequest(page, query, filters))
+//            .awaitSuccess()
+//            .let { response ->
+//                searchAnimeParse(response, query)
+//            }
+//    }
+//
+//    private fun searchAnimeParse(response: Response, query: String): AnimesPage {
+//        val document = response.asJsoup()
+//
+//        val animes = document.select(searchAnimeSelector()).map { element ->
+//            searchAnimeFromElement(element)
+//        }
+//
+//        return AnimesPage(animes, false)
+//    }
+//  private fun searchAnimeSelector(query: String): String = "null"
+
+    override fun searchAnimeRequest(page: Int, query: String, filters: AnimeFilterList): Request = GET("$baseUrl/search/name/$query")
+
+    override fun searchAnimeFromElement(element: Element): SAnime = popularAnimeFromElement(element)
+
+    override fun searchAnimeSelector(): String = popularAnimeSelector()
+
+    override fun searchAnimeNextPageSelector(): String? = null
+
+    // prosta bez filtrów jak na razie :) są dziury ale to kiedyś sie naprawi hihi. Wystarczy dobrze wyszukać animca i powinno wyszukać.
+
+    // =========================== Anime Details ============================
+    override fun animeDetailsParse(document: Document): SAnime {
+        return SAnime.create().apply {
+            // status = document.selectFirst("div.toggle-content > ul > li:contains(Status)")?.let { parseStatus(it.text()) } ?: SAnime.UNKNOWN // Nie pamietam kiedyś sie to naprawi.
+            description = document.selectFirst("p#animeDesc")?.text()
+            genre = document.select("div.row > div.col-12 > span.badge[href^=/search/name/]").joinToString(", ") {
+                it.text()
+            }
+            author = document.select("div.row > div.col-12:contains(Studio:) > span.badge[href=#]").joinToString(", ") {
+                it.text()
+            }
+        }
+    }
+
+    // ============================== Episodes ==============================
+
+    override fun episodeListRequest(anime: SAnime): Request {
+        val url = baseUrl + anime.url
+        return GET(url, apiHeaders)
+    }
+
+    override fun episodeListParse(response: Response): List<SEpisode> {
```

Comment:
Overriding this function seems unnecessary, use `episodeListSelector` and `episodeFromElement` instead

Reply:
Now I was reminded why I did that, because of course I used `episodeListSelector` and `episodeFromElement` before, but it turned out that I can't filter them `episodeListSelector` flies through 12 objects where only e.g. 7 have video because the next ones didn't come out but are already `<li>` added to the page.

Reply:
![Zrzut ekranu 2024-05-23 003002](https://github.com/aniyomiorg/aniyomi-extensions/assets/73482111/0164f799-ae8f-492e-99e9-92e127e00696)